opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -119,8 +119,8 @@ message Span {
   //
   // Bits 8 and 9 represent the 3 states of whether a span's parent
   // is remote. The states are (unknown, is not remote, is remote).
-  // To read whether the value is known, use `flags & SPAN_FLAGS_CONTEXT_HAS_IS_REMOTE_MASK`.
-  // To read whether the span is remote, use `flags & SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK`.
+  // To read whether the value is known, use `(flags & SPAN_FLAGS_CONTEXT_HAS_IS_REMOTE_MASK) >> 8`.
+  // To read whether the span is remote, use `(flags & SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK) >> 9`.
```

Comment:
> I don't understand the motivation. Advising people to use magic constants going against the idea of defining bitmask constants.

I suppose the equivalent is just 
```
boolean hasRemote = (flags & SPAN_FLAGS_CONTEXT_HAS_IS_REMOTE_MASK) != 0;
boolean isRemote = (flags & SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK) != 0;
```
Was just trying to give some additional help on how to use these, but maybe its obvious already.

Reply:
`!= 0` is much better

Reply:
Adjusted